src/constants/index.ts

```diff
@@ -25,64 +25,116 @@ export const EndScene1 = "EndScene1";
+  "./assets/audios/Disapointed-05.mp3";
 export const AUDIO_PATH_POINTS_ADD = "assets/audios/PointsAdd.wav";
-export const AUDIO_PATH_CORRECT_STONE = "assets/audios/CorrectStoneFinal.mp3"
-export const AUDIO_PATH_CHEERING_FUNC = (randomNumber) => `./assets/audios/Cheering-0${randomNumber}.mp3`;
-export const AUDIO_PATH_BTN_CLICK = "./assets/audios/ButtonClick.mp3"
-export const AUDIO_PATH_ON_DRAG = "./assets/audios/onDrag.mp3"
+export const AUDIO_PATH_CORRECT_STONE = "assets/audios/CorrectStoneFinal.mp3";
+export const AUDIO_PATH_CHEERING_FUNC = (randomNumber) =>
+  `./assets/audios/Cheering-0${randomNumber}.mp3`;
+export const AUDIO_PATH_BTN_CLICK = "./assets/audios/ButtonClick.mp3";
+export const AUDIO_PATH_ON_DRAG = "./assets/audios/onDrag.mp3";
+export const AUDIO_ARE_YOU_SURE = "./assets/audios/are-you-sure.mp3";
+export const AUDIO_TIMEOUT = "./assets/audios/timeout.mp3";
+export const AUDIO_INTRO = "./assets/audios/intro.mp3";
+export const AUDIO_LEVEL_LOSE = "./assets/audios/LevelLoseFanfare.mp3";
+export const AUDIO_LEVEL_WIN = "./assets/audios/LevelWinFanfare.mp3";
+export const AUDIO_URL_PRELOAD = [
+  "./assets/audios/intro.mp3",
+  "./assets/audios/Cheering-02.mp3",
+  "./assets/audios/Cheering-03.mp3",
+  "./assets/audios/Cheering-01.mp3",
+  "./assets/audios/onDrag.mp3",
+  "./assets/audios/timeout.mp3",
+  "./assets/audios/LevelWinFanfare.mp3",
+  "./assets/audios/LevelLoseFanfare.mp3",
+  "./assets/audios/ButtonClick.mp3",
+  "./assets/audios/Monster Spits wrong stones-01.mp3",
+  "./assets/audios/Disapointed-05.mp3",
+  "./assets/audios/MonsterSpit.mp3",
+  "./assets/audios/Eat.mp3",
+  "./assets/audios/PointsAdd.wav",
+  "./assets/audios/are-you-sure.mp3",
+];
 
 //Image paths
 export const ASSETS_PATH_STONE_PINK_BG = "./assets/images/stone_pink.png"; //stone_pink_v02.png
-export const ASSETS_PATH_TOTEM = "./assets/images/Totem_v02_v01.png";
-export const ASSETS_PATH_BG_01 = "./assets/images/bg_v01.jpg";
-export const ASSETS_PATH_HILL = "./assets/images/hill_v01.png";
-export const ASSETS_PATH_FENCE = "./assets/images/fence_v01.png";
+export const ASSETS_PATH_TOTEM = "./assets/images/Totem_v02_v01.webp";
+export const ASSETS_PATH_BG_01 = "./assets/images/bg_v01.webp";
+export const ASSETS_PATH_HILL = "./assets/images/hill_v01.webp";
+export const ASSETS_PATH_FENCE = "./assets/images/fence_v01.webp";
 export const ASSETS_PATH_MONSTER_IDLE = "./assets/images/idle4.png";
-export const DEFAULT_BACKGROUND_1 = "./assets/images/bg_v01.jpg";
-export const HILL_IMAGE_1 = "./assets/images/hill_v01.png";
-export const PILLAR_IMAGE_1 = "./assets/images/Totem_v02_v01.png";
-export const FENCE_IMAGE_1 = "./assets/images/fence_v01.png";
-export const AUTUMN_BACKGROUND_1 = "./assets/images/Autumn_bg_v01.jpg";
+export const DEFAULT_BACKGROUND_1 = "./assets/images/bg_v01.webp";
+export const AUTUMN_BACKGROUND_1 = "./assets/images/Autumn_bg_v01.webp";
 export const AUTUMN_HILL_1 = "./assets/images/Autumn_hill_v01.png";
```

Comment:
Question, why is this still in png while the other png are converted to webp?

Reply:
@bernhardccodev I updated the asset. thank you for pointing it out.